src/platform/site-wide/drupal-static-data/tests/connect/fetch.unit.spec.js

```diff
@@ -0,0 +1,39 @@
+import environment from 'platform/utilities/environment';
+
+const sinon = require('sinon');
+const { expect } = require('chai');
+const { fetchDrupalStaticDataFile } = require('../../connect/fetch');
+
+describe('fetchDrupalStaticDataFile', () => {
+  let mockFetch;
+
+  const jsonOK = body =>
+    new Response(JSON.stringify(body), {
+      status: 200,
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+
+  beforeEach(() => {
+    mockFetch = sinon.stub(global, 'fetch').resolves(jsonOK(['test data']));
+  });
+
+  afterEach(() => {
+    global.fetch.restore();
```

Comment:
**CONSIDER**
Is it potentially more clear to call `restore()` on the stub instance (`mockFetch`)?